com.synology.SynologyDrive.yaml

```diff
@@ -31,7 +31,8 @@ modules:
       - type: script
         dest-filename: apply_extra
         commands:
-          - bsdtar -Oxf synology-drive.deb 'data.tar.*' | bsdtar -xf - --strip-components=3 --exclude='./usr'
+          - bsdtar -Oxf synology-drive.deb 'data.tar.*' | bsdtar -xf - --strip-components=3
+            --exclude='./usr'
```

Comment:
Would this new line be a problem?
Otherwise I have no problem bumping the version

Reply:
I don't think so. The bot sometimes reformats things according to new linting rules etc. I've seen it do it with things like single and double quotes, so I think this might just be formatting.